src/webgl/loading.js

```diff
@@ -109,26 +112,47 @@ p5.prototype.loadModel = function(path) {
     failureCallback = arguments[2];
   }
 
+  var fileType = path.slice(-4);
   var model = new p5.Geometry();
   model.gid = path + '|' + normalize;
   var self = this;
-  this.loadStrings(
-    path,
-    function(strings) {
-      parseObj(model, strings);
 
-      if (normalize) {
-        model.normalize();
-      }
+  if (fileType === '.stl') {
```

Comment:
I think if you are going to slice the file extension and test for it, you should probably do the same for `.obj` and then log an error if the file extension isn't either.